common/views/components/GlobalContextProvider/GlobalContextProvider.tsx

```diff
@@ -0,0 +1,92 @@
+import { GetServerSidePropsContext } from 'next';
+import { createContext, FunctionComponent, ReactNode } from 'react';
+import OpeningTimesContext from '../OpeningTimesContext/OpeningTimesContext';
+import GlobalAlertContext from '../GlobalAlertContext/GlobalAlertContext';
+import PopupDialogContext from '../PopupDialogContext/PopupDialogContext';
+import TogglesContext from '../TogglesContext/TogglesContext';
+import { parseCollectionVenues } from '../../../services/prismic/opening-times';
+
+export type GlobalContextData = {
+  toggles: any;
+  globalAlert: any;
+  popupDialog: any;
+  openingTimes: any;
+};
+
+type Props = {
+  value: GlobalContextData;
+  children?: ReactNode;
+};
+
+const defaultValue = {
+  toggles: {},
+  globalAlert: null,
+  popupDialog: null,
+  openingTimes: null,
+};
+
+export type WithGlobalContextData = {
+  globalContextData: GlobalContextData;
+};
+
+const Context = createContext<GlobalContextData>(defaultValue);
+const GlobalContextProvider: FunctionComponent<Props> = ({
+  value,
+  children,
+}: Props) => {
+  const parsedOpeningTimes = parseCollectionVenues(value.openingTimes);
+  return (
+    <Context.Provider value={value}>
+      <OpeningTimesContext.Provider value={parsedOpeningTimes}>
+        <GlobalAlertContext.Provider value={value.globalAlert}>
+          <PopupDialogContext.Provider value={value.popupDialog}>
+            <TogglesContext.Provider value={value.toggles}>
+              {value.toggles.helveticaRegular && (
+                <style
+                  type="text/css"
+                  dangerouslySetInnerHTML={{
+                    __html: `
+                      @font-face {
+                        font-family: 'Helvetica Neue Light Web';
+                        src: local('Helvetica Neue Regular'),
```

Comment:
Something for us to think about (not in this pr obvz):  https://www.bramstein.com/writing/web-font-anti-patterns-local-fonts.html